### PR TITLE
CLI: manage config path and log level with global flags

### PR DESCRIPTION
**Pull request by Nexucis**
this PR introduced the usage of global flags to be able to set the path to the config file and the level of the log.

This PR must be merged after #314

### CODE REVIEW

File: cmd/percli/main.go [hunk 1]
```diff
@@ -21,21 +21,37 @@ import (
...
+	// The list of the commands supported
 	cmd.AddCommand(describe.NewCMD())
 	cmd.AddCommand(get.NewCMD())
 	cmd.AddCommand(login.NewCMD())
 	cmd.AddCommand(project.NewCMD())
 	cmd.AddCommand(version.NewCMD())
+
+	// the list of the global flags supported
+	cmd.PersistentFlags().StringVar(&configPath, "percliconfig", cmdUtils.GetDefaultConfigPath(), "Path to the percliconfig file to use for CLI requests.")
+	cmd.PersistentFlags().StringVar(&logLevel, "log.level", "info", "Set the level of the log. Possible value: panic, fatal, error, warning, info, debug, trace")
```
Comment (juliusv) [inline cmd/percli/main.go:49]:
```suggestion
	cmd.PersistentFlags().StringVar(&logLevel, "log.level", "info", "Set the log verbosity level. Possible values: panic, fatal, error, warning, info, debug, trace")
```

File: cmd/percli/main.go [hunk 2]
```diff
@@ -46,14 +62,22 @@ func initLogrus() {
 		// https://github.com/sirupsen/logrus/issues/896
 		FullTimestamp: true,
 	})
+	level, err := logrus.ParseLevel(logLevel)
+	if err != nil {
+		logrus.WithError(err).Fatal("unable to set the log.level")
```
Comment (juliusv) [inline cmd/percli/main.go:67]:
```suggestion
		logrus.WithError(err).Fatal("unable to set the log level")
```

File: internal/cli/utils/config.go [hunk 3]
```diff
@@ -87,13 +93,14 @@ func getRootFolder() string {
 
 // ReadConfig reads the configuration file stored in the path {USER_HOME}/.argos/config
 // If there is no error during the read, it returns the result in the struct ArgosCLIConfig
-func readConfig() (*CLIConfig, error) {
-	path := filepath.Join(getRootFolder(), pathConfig, configFileName)
-	if _, err := os.Stat(path); os.IsNotExist(err) {
-		return nil, fmt.Errorf("file %s doesn't exist", path)
+func readConfig(filePath string) (*CLIConfig, error) {
+	if _, err := os.Stat(filePath); os.IsNotExist(err) {
+		return nil, fmt.Errorf("file %s doesn't exist", filePath)
```
Comment (juliusv) [inline internal/cli/utils/config.go:98]:
```suggestion
		return nil, fmt.Errorf("file %q doesn't exist", filePath)
```

**Review by juliusv — COMMENTED** [review]
:+1: some nits :)

**Comment by Nexucis** [comment]
thanks @juliusv !

**Pull request merged**